qtpy/QtSerialBus.py

```diff
@@ -0,0 +1,30 @@
+# -----------------------------------------------------------------------------
+# Copyright © 2009- The Spyder Development Team
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
```

Comment:
We should at least be consistent about whether we're adding this header to the tests—either to all newly added ones (ideal), or else none.

Reply:
Is this the correct copyright header? Various files have slightly different headers and some of them also list specific individuals, so I've never been sure what header I should be adding.

Reply:
I was going to comment on that, but I didn't want to be too nitpicky, heh.

The up to date version of the canonical header (for new files, at least) is [in the License Guidelines on the our governance repo](https://github.com/spyder-ide/governance-and-guidelines/blob/main/license_guidelines.md#license-header), i.e.:

```suggestion
# -----------------------------------------------------------------------------
# Copyright (c) 2023- QtPy contributors
#
# Released under the terms of the MIT License
# (see LICENSE.txt in the project root directory for details)
# -----------------------------------------------------------------------------
```

The reason a number of files list different individuals is because a lot of the code was combined from multiple predecessor projects (see the [Readme section](https://github.com/spyder-ide/qtpy/tree/master#attribution-and-acknowledgments) that touches on a bit of this history), plus various other historical contributions. New files should just use the above, but it is important to retain the existing copyright notices in files that have them for both legal and ethical purposes.